kevm

```diff
@@ -103,21 +103,30 @@ run_kast() {
 }
 
 run_prove() {
-    local def_module run_dir proof_args bug_report_name
+    local def_module run_dir proof_args haskell_backend_args bug_report_name eventlog_name
 
     check_k_install
 
     def_module="$1" ; shift
     bug_report_name="kevm-bug-$(basename "${run_file%-spec.k}")"
+    eventlog_name="$run_file.log.json"
+
+    haskell_backend_args=(initial)
+    ! $bug_report || haskell_backend_args+=(--bug-report "$bug_report_name")
+    ! $profile    || haskell_backend_args+=(+RTS -l -ol$eventlog_name -RTS)
+    haskell_backend_args="${haskell_backend_args[@]:1}"
```

Comment:
I would instead start the `haskell_backend_args=(kore-exec)` as the intiial array, and add to it. Then later, have a check which looks like `[[ "${#haskell_backend_args}" -le 1 ]] || proof_args+=(--haskell-backend-command "${haskell_backend_args}")`